src/redis-cli.c

```diff
@@ -1936,6 +1945,8 @@ static void usage(int err) {
 "  --replica          Simulate a replica showing commands received from the master.\n"
 "  --rdb <filename>   Transfer an RDB dump from remote server to local file.\n"
 "                     Use filename of \"-\" to write to stdout.\n"
+" --functions-rdb <filename> Like --rdb but only get the functions (not the keys)\n"
+"                     when getting the RDB dump file.\n"
```

Comment:
```suggestion
" --functions-rdb <filename> Like --rdb but only get the functions (not the keys)\n"
"                            when getting the RDB dump file.\n"
```

Reply:
Was not sure about this, for arguments that are too long I see two different approaches in the current code:
Align second line according to all other arguments:
https://github.com/redis/redis/blob/b0c06e904a94bbc2da8003a1b230482ee34d6d5f/src/redis-cli.c#L1956-L1957
Align second line according to first line:
https://github.com/redis/redis/blob/b0c06e904a94bbc2da8003a1b230482ee34d6d5f/src/redis-cli.c#L1954-L1955
I like the second approach better and since redis-cli isn't consistent that's what I chose for `--functions-rdb`.

Reply:
Because when redis-cli output, if use the second way, we can know the command on the left and the comment on the right at a glance, while the first way requires more glances.